ocaml/runtime/fiber.c

```diff
@@ -49,13 +49,16 @@
 
 static _Atomic int64_t fiber_id = 0;
 
-uintnat caml_get_init_stack_wsize (void)
+uintnat caml_get_init_stack_wsize (bool is_main_thread)
 {
-  uintnat default_stack_wsize = Wsize_bsize(Stack_init_bsize);
+  uintnat init_stack_wsize =
```

Comment:
As discussed, we should use the normal OCaml 5 logic when stack checks are enabled (same everywhere in this PR).

Reply:
(Added `#ifdef`s where necessary.)